lib/utils/uniqueID.js

```diff
@@ -0,0 +1,9 @@
+var index = 0
+
+// This is of course not using UUIDs, but should be totally sufficient
+// for our use case
+function uniqueID () {
+  return 'faux-dom-' + (index++)
+}
```

Comment:
> I mean, you can do that but you'll have new ids every time so it'll be slow.

Are we sure about that? Is it going to be slower? Anyway. Maybe there is a more deterministic method of creating the keys instead of using a global counter. Though I don't think there is a stateless way, considering my next point ...

> I just don't like the statefulness of it, personally.

I do understand this. In your use-case the statefulness is not really necessary. In other user-cases, where faux DOM nodes are moved around, there is not way around it...